### PR TITLE
Make minor icon related improvements

### PR DESCRIPTION
**Pull request by alrra**
* <del>Use a `192×192px` touch icon, as that is the [recommended size](https://developer.chrome.com/multidevice/android/installtohomescreen#best-practices) for Android.</del> (see [comment](https://github.com/operasoftware/devopera/pull/312#discussion_r33857961)).
   
* Use a `180×180px` Apple touch icon:
    
    * Add a `180×180px` `apple-touch-icon` as that is the [recommended size](  https://github.com/h5bp/html5-boilerplate/commit/345925f04384b59929d8d39ba428cef548921408) since the iOS 8 release.

    * <del>Include `<link rel="apple-touch-icon" href="apple-touch-icon.png">` as Apple usually changes what is requested by default (see: https://github.com/h5bp/html5-boilerplate/issues/1615 &      https://github.com/h5bp/html5-boilerplate/pull/1622)</del> (see [comment](https://github.com/operasoftware/devopera/pull/312#issuecomment-118023788)).

--

Cc: @mathiasbynens

### CODE REVIEW

**Comment by alrra** [comment]
Cc: @mathiasbynens

**Comment by mathiasbynens** [comment]
LGTM, but I’d rather not add `<link rel="apple-touch-icon" href="apple-touch-icon.png">`. Apple should implement Web Manifest support.

**Comment by alrra** [comment]
> but I’d rather not add <link rel="apple-touch-icon" href="apple-touch-icon.png">. Apple should implement Web Manifest support.

@mathiasbynens Change made. 

--

Note: Until Apple decides to implement the web manifest, you're serving two 404 pages [1] until the image is actually requested. 

On iOS 8 the requests made are the following:

* /apple-touch-icon-152x152-precomposed.png 
* /apple-touch-icon-152x152.png 
* /apple-touch-icon.png

File: devopera.webmanifest [hunk 1]
```diff
@@ -3,8 +3,8 @@
 	"name": "Dev.Opera",
 	"icons": [
 		{
-			"src": "apple-touch-icon-144x144.png",
-			"sizes": "144x144",
+			"src": "touch-icon.png",
```
Comment (mathiasbynens) [inline devopera.webmanifest]: Let’s keep the `apple-touch-icon.png` name and reference that here.

Comment (alrra) [inline devopera.webmanifest]: > Let’s keep the apple-touch-icon.png name and reference that here.

@mathiasbynens The `apple-touch-icon.png` right now is only `180×180px`, while the recommended size for Android is [`192×192px`](https://developer.chrome.com/multidevice/android/installtohomescreen#best-practices). 

Do you want me to just use one `192×192px` image named `apple-touch-icon.png`?

Comment (alrra) [inline devopera.webmanifest]: > Let’s keep the apple-touch-icon.png name and reference that here.

@mathiasbynens Done.

**Comment by mathiasbynens** [comment]
Thanks @alrra!

**Pull request merged**